targeteditor/src/main/scala/explore/targeteditor/ElevationPlot.scala

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.targeteditor
+
+import java.time.Duration
+import java.time.Instant
+
+import scala.scalajs.js
+
+import cats.implicits._
+import explore.model.reusability._
+import gem.enum.Site
+import gpp.highcharts.highchartsStrings.line
+import gpp.highcharts.mod.AxisTypeValue
+import gpp.highcharts.mod.ChartOptions
+import gpp.highcharts.mod.CreditsOptions
+import gpp.highcharts.mod.LegendOptions
+import gpp.highcharts.mod.Options
+import gpp.highcharts.mod.PlotOptions
+import gpp.highcharts.mod.PlotSeriesOptions
+import gpp.highcharts.mod.Point
+import gpp.highcharts.mod.PointMarkerOptionsObject
+import gpp.highcharts.mod.PointOptionsObject
+import gpp.highcharts.mod.SeriesLineOptions
+import gpp.highcharts.mod.SeriesOptionsType
+import gpp.highcharts.mod.SeriesStatesHoverOptionsObject
+import gpp.highcharts.mod.SeriesStatesOptionsObject
+import gpp.highcharts.mod.TitleOptions
+import gpp.highcharts.mod.TooltipOptions
+import gpp.highcharts.mod.TooltipShapeValue
+import gpp.highcharts.mod.XAxisOptions
+import gpp.highcharts.mod.YAxisLabelsOptions
+import gpp.highcharts.mod.YAxisOptions
+import gpp.highcharts.mod.YAxisTitleOptions
+import gpp.ui.reusability._
+import gsp.math.Angle
+import gsp.math.Coordinates
+import gsp.math.skycalc.ElevationCalc
+import japgolly.scalajs.react._
+import react.common._
+import react.highcharts.Chart
+
+import js.JSConverters._
+
+final case class ElevationPlot(
+  site:   Site,
+  coords: Coordinates
+) extends ReactProps[ElevationPlot](ElevationPlot.component)
+
+object ElevationPlot {
+  type Props = ElevationPlot
+
+  implicit val propsReuse: Reusability[Props] = Reusability.derive
+
+  val PlotPeriod: Duration = Duration.ofHours(12)
+  val PlotEvery: Duration  = Duration.ofMinutes(1)
+
+  class Backend( /*$ : BackendScope[Props, State]*/ ) {
+    def render(props: Props) = {
+
+      val start      = Instant.now()
+      val end        = start.plus(PlotPeriod)
+      val elevations =
+        ElevationCalc.elevationsForInterval(props.site, start, end, PlotEvery, _ => props.coords)
+
+      val options = Options()
+        .setChart(ChartOptions().setHeight(280))
```

Comment:
I suppose this come from the generated facade. if we were to do this by hand, the height would be a field right?

Reply:
Highcharts is supposed to adapt to the height of the container, but I can't get that work yet in our setup and it was showing it's default height of 400px. This is meant to be temporary for demo purposes. Worst case, I'm counting on `SizeMe`.